### DISCUSSION TITLE
Docstrings, task list

### DISCUSSION THREAD

**Discussion by razofz** [parashardhapola/scarf#45] [Ideas]
List of docstrings that are currently missing:

- `writers.py`

  - [ ] `write` function in class `ZarrMerge`
  - [ ] all `dump` methods

- `assay.py`

  - [x] description in module level docstring
  - [x] `{ADT,ATAC}assay`
  - [x] the `norm_*` methods
  - [x] functions in `{RNA,ADT}assay` (e. g. `normed` and `set_feature_stats`)

- `meld_assay.py`

  - [ ] description in module level docstring
  - [ ] function `meld_assay`

- `plots.py`

  - [ ] functions `plot{_graph,}_qc`, Are these for qc of a graph, or for plotting a qc plot?

- `readers.py`

  - [x] `H5adReader`, description for attribute `groupCodes`

- `metadata.py`

  - [ ] several methods in class `MetaData`
  
- `ann.py`

  - [ ] module level docstring
  - [ ] plus everything in it

- `dendrogram.py`

  - [ ] module level docstring
  - [ ] `CoalesceTree`
  - [ ] `BalancedCut`
 
- `plots.py`

  - [ ] most of the methods
  
- `markers.py`

  - [ ] `find_markers_by_rank`

 - `umap.py`

   - [ ] no docstrings here.

**Comment by parashardhapola** [comment]
This is fantastic. Thanks @razofz.
